chain/network/src/test_utils.rs

```diff
@@ -219,23 +219,6 @@ impl Handler<GetInfo> for PeerManagerActor {
     }
 }
 
-/// `GetBroadcastMessageCount` gets `NetworkMetrics` from `PeerManager`.
```

Comment:
Always happy to see deadcode being removed.  Would it be better to split unrelated changes into separate PRs?

Reply:
The removed deadcode is a getter of an unused metric, so it is related.